### DISCUSSION TITLE
How do I execute build commands inside CICD?

### DISCUSSION THREAD

**Discussion by warrenronsiek** [microsoft/react-native-windows#13306] [Q&A] [answered]
The documentation says that you should do builds via running `react-native run-windows`. This works, however it also launches the program, which is inconvenient inside of a CICD environment where I just want to do the build and then exit. 

I've tried doing vanilla `msbuild` commands, but these fail, presumably because `react-native run-windows` does some necessary legwork first. Is there some standard way I can do a react native build without launching the application?

**Comment by chrisglein** [comment, marked as answer]
There should be a `no-launch` which will prevent it from running, or `no-deploy` if you don't even want to install it. Do those work for you?

https://microsoft.github.io/react-native-windows/docs/run-windows-cli

Also there are more docs on setting up CI here: https://microsoft.github.io/react-native-windows/docs/setup-ci
Note pending changes: https://github.com/microsoft/react-native-windows-samples/pull/942
